File: app/src/main/java/org/deiverbum/app/model/crud/CrudToday.java
package org.deiverbum.app.model.crud;

import org.deiverbum.app.model.Today;

import java.util.ArrayList;
import java.util.List;

public class CrudToday {

    public List<Today> c;
    public List<Today> u;
    public List<Today> d;

    @SuppressWarnings("unused")
    public CrudToday() {
    }

    public void c(Today t) {
        if (c == null) {
            c = new ArrayList<>();
        }
        c.add(t);
    }

    public void addUpdate(Today t) {
        if (u == null) {
            u = new ArrayList<>();
        }
        u.add(t);
    }

    public void addCreate(Today t) {
        if (c == null) {
            c = new ArrayList<>();
        }
        c.add(t);
    }

    public void addDelete(Today t) {
        if (d == null) {
            d = new ArrayList<>();
        }
        d.add(t);
    }


}

File: app/src/main/java/org/deiverbum/app/core/model/data/crud/CrudToday.kt
package org.deiverbum.app.core.model.data.crud

import org.deiverbum.app.core.model.data.Today

class CrudToday {
    var c: MutableList<Today>? = null
    var u: MutableList<Today>? = null
    var d: MutableList<Today>? = null
    fun c(t: Today) {
        if (c == null) {
            c = ArrayList()
        }
        c!!.add(t)
    }

    fun addUpdate(t: Today) {
        if (u == null) {
            u = ArrayList()
        }
        u!!.add(t)
    }

    fun addCreate(t: Today) {
        if (c == null) {
            c = ArrayList()
        }
        c!!.add(t)
    }

    fun addDelete(t: Today) {
        if (d == null) {
            d = ArrayList()
        }
        d!!.add(t)
    }
}